Presentation.Web/app/components/it-system/edit/it-system-edit.controller.ts

```diff
@@ -116,12 +117,16 @@
                             $state.go('it-system.catalog');
                         })
                         .error(function (data, status) {
-                            if (status === 409)
-                                msg.toErrorMessage('Fejl! IT Systemet er i lokal anvendelse!');
-                            else if (status === 401)
-                                msg.toErrorMessage('Fejl! Du har ikke tilladelse!');
-                            else
-                                msg.toErrorMessage('Fejl! Kunne ikke slette IT System!');
+                            if (status === 401) {
+                                msg.toErrorMessage(errormsg.errorMessageNoPermission);
+                            }
+                            else if (status === 409) {
+                                msg.toErrorMessage(Kitos.Utility.ToastTranslatorTool.translateItSystemDeletionConflictResponse(data.response));
```

Comment:
Kan du ikke lave en import af modulet i stedet for fully qualified name?
https://www.typescriptlang.org/docs/handbook/modules.html